pep-8016.rst

```diff
@@ -291,14 +299,19 @@ maintain Python, core team members who have stopped contributing are
 encouraged to declare themselves as "emeritus members". Those who
 haven't made any non-trivial contribution in two years may be asked to
 move themselves to this category, and moved there if they don't
-respond. Emeritus team members lose their privileges such as voting
-rights and commit access. If an emeritus team member later resumes
-contributing, they may rejoin the core team on request, without the
-need for a vote.
+respond. To record and honor their contributions, emeritus team
+members will continue to be listed alongside active core team members;
+and, if they later resume contributing, they may switch themselves
+back to active status on request, without the need for a vote. While
```

Comment:
"without the need for a vote" why do you mention a (lack of) vote here? There is no vote to move a core dev to the Emeritus list, no? (I'm now confused.)

Reply:
It's intended to say: if someone wants to switch from Emeritus back to active, then they don't have to go through the whole vote-to-become-a-core-team-member thing again. If that wasn't clear then I guess I better rephrase some more :-)

Reply:
Oh, I didn't think about the vote to promote a core developer. Now it makes sense.

Note: you may also mention that keeping inactive core developers is threat for Python security.